README.rst

```diff
@@ -28,30 +33,8 @@ country data::
     crash < sql/schemas.sql
     crash -c "COPY guestbook.countries FROM '$(pwd)/sql/countries.json' RETURN SUMMARY;"
 
-If you choose to use Docker for running CrateDB on your workstation, use those
-commands::
-
-    # Define CrateDB version.
-    export CRATEDB_VERSION=latest
-    export CRATEDB_IMAGE=crate:${CRATEDB_VERSION}
-
-    # Start CrateDB.
-    docker run -it --rm \
-        --volume=$(pwd)/sql:/sql \
-        --publish=4200:4200 --publish=5432:5432 \
-        ${CRATEDB_IMAGE} \
-        -Cdiscovery.type=single-node \
-        -Ccluster.routing.allocation.disk.threshold_enabled=false
-
-    # Populate schema and data, and verify it's there.
-    docker run --rm --network=host --volume=$(pwd)/sql:/sql ${CRATEDB_IMAGE} sh -c "$(cat <<EOT
-        crash < /sql/schemas.sql;
-        crash <<SQL
-            COPY guestbook.countries FROM 'file:///sql/countries.json' RETURN SUMMARY;
-            REFRESH TABLE guestbook.countries;
-            SELECT id, name FROM guestbook.countries LIMIT 10;
-    SQL
-    EOT)"
+Please note that the ``countries.json`` file needs to be accessible by CrateDB.
```

Comment:
This is also a problem for standalone installations and tutorials where the data is not publicly available on S3, but is individually provided instead. I would love to see a part of the outcome of the current Cloud endeavor to also be available for other users.

- https://github.com/crate/cloud/issues/1026
- https://github.com/crate/stevedore

I can imagine a poor man's version by using [`rclone`](https://rclone.org/) to quickly upload a file to an S3 storage [^1], and then importing the data from there. It may even be wrapped up into a little helper program for that very purpose.

[^1]: Do you know of any _publicly_ available S3 stores, which can be used _anonymously_ for _temporarily_ storing files?

/cc @SStorm, @seut, @proddata, @hammerhead, @marijaselakovic, @hlcianfagna

Reply:
I just tried
```
COPY guestbook.countries FROM 'https://raw.githubusercontent.com/crate/crate-sample-apps/main/sql/countries.json';
```
and it worked for me

Reply:
Ah sure, just using a plain HTTP upload target / acquisition source for temporarily storing that file would work equally well. So, any publicly available and anonymously writable web space would also work. Do you know any?

Reply:
> Any publicly available and anonymously writable web space would also work. Do you know any?

I've tried to find a suitable cyberlocker aka. [file-hosting service], but it turns out to be harder than expected.

Most of the services obfuscate the download step these days, by placing an interstitial HTML page - very sad. Within the list of [alternatives to WeTransfer], I've finally discovered [Oshi - Anonymous file storage], which seems to be the only software available [^1] to offer a command line interface based on `curl`, see [Oshi command-line reference].

dcbfdee9d0 has a corresponding improvement to the documentation, how to use it to upload the `sql/countries.json` file.
```
countries_url=$(curl --silent --header 'X-Requested-With: XMLHttpRequest' \
   --form f=@sql/countries.json https://oshi.at | jq -r '.files[0].url')
crash -c "COPY guestbook.countries FROM '${countries_url}' RETURN SUMMARY;"
```


[alternatives to WeTransfer]: https://alternativeto.net/software/wetransfer/
[file-hosting service]: https://en.wikipedia.org/wiki/File_hosting_service
[Oshi - Anonymous file storage]: https://oshi.at/
[Oshi command-line reference]: https://oshi.at/cmd

[^1]: Also with sources, see https://github.com/somenonymous/OshiUpload.

Reply:
The Oshi option looks good. For the sake of simplicity, we could also go with the GitHub option Hernan mentioned earlier (i.e. copying into CrateDB directly from https://raw.githubusercontent.com/crate/crate-sample-apps/main/sql/countries.json)? For the low traffic we expect this should be acceptable I think, I've seen many other projects taking that approach as well for installer scripts or other small provisioning payloads.

Reply:
> For the sake of simplicity...

I agree. 7040a5e6eef9 implements the suggestion by @hlcianfagna. Thanks a stack!